dask_jobqueue/__init__.py

```diff
@@ -1,15 +1,15 @@
-# flake8: noqa
-from . import config
-from .core import JobQueueCluster
-from .moab import MoabCluster
-from .pbs import PBSCluster
-from .slurm import SLURMCluster
-from .sge import SGECluster
-from .lsf import LSFCluster
-from .oar import OARCluster
-from .htcondor import HTCondorCluster
-
-from ._version import get_versions
-
-__version__ = get_versions()["version"]
-del get_versions
+# flake8: noqa
+from . import config
+from .core import JobQueueCluster
+from .moab import MoabCluster

```

Comment:
I'm not sure I understand what has changed in this file, and I see no ConductorCluster?